test/unit/server-securitycontextconstraints.bats

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bats
+
+load _helpers
+
+@test "server/SecurityContextConstraints: disabled by default" {
+  cd `chart_dir`
+  assert_empty helm template \
+      -s templates/server-securitycontextconstraints.yaml  \
+      .
+}
+
+@test "server/SecurityContextConstraints: disabled with server disabled and global.openshift.enabled=true" {
+  cd `chart_dir`
+  assert_empty helm template \
+      -s templates/server-securitycontextconstraints.yaml  \
+      --set 'server.enabled=false' \
+      --set 'global.openshift.enabled=true' \
+      .
+}
+
+@test "server/SecurityContextConstraints: enabled with global.openshift.enabled=true and server.exposeGossipAndRPCPorts=true" {
+  cd `chart_dir`
+  local actual=$(helm template \
+      -s templates/server-securitycontextconstraints.yaml  \
+      --set 'global.openshift.enabled=true' \
+      --set 'server.exposeGossipAndRPCPorts=true' \
+      . | tee /dev/stderr |
+      yq -s 'length > 0' | tee /dev/stderr)
+  [ "${actual}" = "true" ]
+}
+
+@test "server/SecurityContextConstraints: host ports are allowed by default" {
+  cd `chart_dir`
+  local actual=$(helm template \
+      -s templates/server-securitycontextconstraints.yaml  \
+      --set 'global.openshift.enabled=true' \
+      --set 'server.exposeGossipAndRPCPorts=true' \
+      . | tee /dev/stderr |
+      yq -c '.allowHostPorts' | tee /dev/stderr)
+  [ "${actual}" = 'true' ]
+}
+
```

Comment:
```suggestion
}

```

Since this isn't templated we don't need a test for it. We try and only add tests for things that change based on an if/else.